src/subscription_manager/gui/data/glade/factsdialog.glade

```diff
@@ -4,7 +4,7 @@
   <!--
  interface-naming-policy project-wide -->
   <object class="GtkWindow" id="system_facts_dialog">
-    <property name="width_request">500</property>
+    <property name="width_request">550</property>
```

Comment:
I'm suggesting that we do both: go ahead and adjust the width, and also wrap. That way, ideally, wrapping won't be necessary, but in the case it is, the text will still be visible.